src/login.rs

```diff
@@ -481,3 +529,13 @@ impl Login {
         .unwrap()
     }
 }
+
+fn activate_error_label(error_label: &gtk::Label, message: &str) {
```

Comment:
```suggestion
fn show_error_label(error_label: &gtk::Label, message: &str) {
```

`activate` is used in other contexts in gtk, so it may create confusion.